docs/sphinx/source/whatsnew/v0.7.0.rst

```diff
@@ -77,7 +77,7 @@ API Changes
          - `pvsystem.physicaliam` is `iam.physical`
          - `pvsystem.ashraeiam` is `iam.ashrae`
          - `pvsystem.sapm_aoi_loss` is `iam.sapm`
-      - Created dict `iam.IAM_MODEL_PARAMS` to aid in identifying IAM models
+      - Created dict `iam._IAM_MODEL_PARAMS` to aid in identifying IAM models
```

Comment:
I think we can delete this line entirely. No need to advertise the private dictionary's existence.

Reply:
Yeah makes sense, will drop it.